config/esp32/components/chip/Kconfig

```diff
@@ -47,6 +47,13 @@ menu "CHIP Core"
                 A Binding object is used to configure how the local device communicates with
                 a remote entity, be it a cloud service, a mobile application, or another device.
 
+        config MAX_DEVICE_ADMINS
+            int "Max device admins"
+            range 0 16
```

Comment:
The spec-allowed range for this value is 5-255.

Reply:
Changed